tests/robot-tests/tests/admin/bau/create_methodology_for_publication.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Resource    ../../libs/admin-common.robot
+Library  ../../libs/admin_api.py
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       user signs in as bau1
+Suite Teardown    teardown suite
```

Comment:
Rather than declaring a `teardown suite` keyword, you could remove that and use 
```
Suite Teardown   user closes the browser
```

Reply:
Done ta